libtorch_test/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
The libtorch_test files should not be included in the PR since we don't want them on master.

Reply:
Removed